cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDLanding.java

```diff
@@ -0,0 +1,142 @@
+package com.skillstorm.StepDefinitions;
+
+import java.time.Duration;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.bidi.log.Log;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import com.skillstorm.WebDriverSingleton;
+import com.skillstorm.PageObjects.LandingPage;
+import com.skillstorm.PageObjects.LoginPage;
+import com.skillstorm.PageObjects.Components.Navbar.LandingNavbar;
+import com.skillstorm.PageObjects.Components.Navbar.Navbar;
+import com.skillstorm.Utilities.Navigator;
+import com.skillstorm.Utilities.UserData.User;
+import com.skillstorm.Utilities.UserData.UserType;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class SDLanding {
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+    private Navigator navigator;
+    private LandingPage page;
+    private User user = new User(UserType.PERSISTANT, "joseph.sam@gmail.com", "password1");
```

Comment:
Medium impact: Potential modifications to given user that other devs may be working with. Suggestion: use a new local user.

Reply:
fixed to use persistent user for now that I can set locally